examples/haproxy/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+services:
+  neginx:
```

Comment:
Not that it matters, but, typo?

Reply:
Yep. Didn't spot it even in my personal configuration. Thanks!